### DISCUSSION TITLE
Multicluster traffic view in Linkerd Viz

### DISCUSSION THREAD

**Discussion by manju-rn** [linkerd/linkerd2#13261] [General]
How to view the traffic across the cluster in Linkerd Viz?
I have the following setup:
1.  2 clusters - east and west connected via linkerd in hierarchical network model.
2.  An adminer pod (a simple UI for DB) hosted in east cluster
3.  A postgres DB (installed as deployment with PV) hosted in west cluster with labels of `mirror.linkerd.io/exported=true` in the respective namespace
4.  In east cluster, the corresponding mirror service is reflected and the Adminer client successfully connects to postgres in west cluster

However, the linkerd viz does not show any metrics or graph which indicates the traffic flows to west cluster?  It only displays the traffic until the adminer app. How can this be achieved to show case traffic across cluster which means showing that it is connecting to the postgres svc in west cluster?

![image](https://github.com/user-attachments/assets/1e80f603-7d3b-4144-893d-d5910e284b15)
![image](https://github.com/user-attachments/assets/2cd40d03-9646-4ec3-8956-5314aaf20536)

I have done same setup in Istio and here is the output from kiali.  Wanted to achieve similar in linkerd _For Info, there is no traces enabled. This is the same app as done in linkerd_
![image](https://github.com/user-attachments/assets/40c944a7-9450-4ce6-ac9f-989de5217f35)

**Comment by magikbits** [comment]
Hi @manju-rn, 
were you able to achive this somehow in linkerd?

**Reply by manju-rn** [reply]
No, I kind of not using linkerd nowadays. I am using cilium as it is sidecar less. Also the hubble view in cilium is terrific, similar to kiali.
And should I say, much easier to install?

**Comment by bezarsnba** [comment]
Hi @manju-rn ,I know this is quite an old topic, but as far as I know, Linkerd Viz is an extension that resides only within individual clusters.

However, there are some proxy metrics that can provide dst_pod and dst_target_cluster references:  https://linkerd.io/2.15/reference/proxy-metrics/#outbound-labels

This is a example PromQL show the information:

```
sum(tcp_open_connections{direction="outbound",namespace="emojivoto",deployment="web"}) by (dst_target_cluster)
```
